app/views/home/index.html.erb

```diff
@@ -12,9 +12,9 @@
         <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum volutpat laoreet risus sed condimentum. Nulla facilisi. Nunc mattis purus in bibendum blandit. Phasellus fermentum erat ultrices nunc aliquam interdum. Mauris rhoncus, nisi quis scelerisque dapibus, lorem ex fringilla felis, et mattis nisl eros at massa. Aenean condimentum tempus ornare. Ut scelerisque nulla ut elit iaculis, nec egestas turpis tincidunt. Suspendisse elementum pharetra auctor. Donec semper imperdiet nisi, sit amet varius sem commodo sed. Pellentesque bibendum euismod risus, luctus iaculis lacus hendrerit dictum. Proin at lectus pharetra, pharetra quam quis, dapibus purus.</p>
       </div>
     </div>
+    <%= render "shared/footer", class: "foot" %>
   <% else %>
     <%= render "shared/cardbuild", class: "cardbuild" %>
-    <%#= render "shared/footer", class: "foot" %>
     <%= render "shared/footer2", class: "foot" %>
   <% end %>
 
```

Comment:
Nice, This is what I', talking about. We can isolate everything into partials so it's easier to check and update if necessary!